githubAPI/octokit.js

```diff
@@ -0,0 +1,58 @@
+const { Octokit } = require("@octokit/rest");
+require('dotenv').config();
+
+const octokit = new Octokit({
+  auth: process.env.WEBHOOK_SECRET,
+  log: {
+    debug: () => {},
+    info: () => {},
+    warn: console.warn,
+    error: console.error
+  }
+});
+
+if(!octokit.auth) {
+  console.log('Missing authentication token. Please add correct environment variable to file .env!');
+  return undefined;
+}
```

Comment:
I think this should probably be wrapped in a function (e.g. a singleton constructor)?

Reply:
You just asks for difficult things! I have no idea how to make a singleton in JS> Ask me to make it in C++ and I'll be happy to do it. : D